include/zephyr/bluetooth/mesh/access.h

```diff
@@ -1070,6 +1070,11 @@ void bt_mesh_model_data_store_schedule(struct bt_mesh_model *mod);
  *  extension list and element, giving the models extended subscription list
  *  capacity.
  *
+ * If @kconfig{CONFIG_BT_MESH_COMP_PAGE_1} is enabled, either base_mod or
```

Comment:
with reference on CDP1 feature? Since without this, it can be called.

Reply:
Ok, fine, leave CDP1 as a reference. This will become default later anyway.

Reply:
Since the extension normally is a part of one of the init callbacks, perhaps:

>  It is not allowed to call this function before the :c:member:`bt_mesh_model_cb.init` callback is called for both models, or it can be called as part of the final callback.

Reply:
Updated!